etserver/getters.py

```diff
@@ -89,8 +90,15 @@
     version = (resp.get("tag_name") or resp.get("name", "Unknown")).lstrip("v")
     project_info["version"] = version
     project_info["status"] = status_code
-    # TODO: query .etelemetry.json to add additional fields to project_info
+
     if status_code == 200:
+        resp_et = requests.get(GITHUB_ET_FILE.format(owner=owner, repo=repo))
```

Comment:
to keep it asynchronous

```suggestion
        resp_et = await fetch_response(app, GITHUB_ET_FILE.format(owner=owner, repo=repo))
```

Reply:
fetch response uses the session, which i don't think is necessary, and was resulting in an error.

Reply:
what was the error? it would be nice to use aiohttp for all requests, to avoid a potential deadlock here

Reply:
it needs some parameters, but it was unclear to me what the params were. i didn't have time to look into sanic's session API

also the response code is not being checked in fetch_response. so if it returns a 404, it still assumes that the body can be checked, so raises an exception.

Reply:
```
et-server  | [2019-11-19 15:30:38 +0000] [1] [ERROR] Exception occurred while handling uri: 'http://localhost:8000/projects/nipy/nipype'
et-server  | Traceback (most recent call last):
et-server  |   File "/usr/local/lib/python3.7/site-packages/sanic/app.py", line 917, in handle_request
et-server  |     response = await response
et-server  |   File "/usr/local/lib/python3.7/site-packages/etserver/serve.py", line 118, in get_project_info
et-server  |     project_info = await fetch_project(app, owner, repo)
et-server  |   File "/usr/local/lib/python3.7/site-packages/etserver/getters.py", line 45, in fetch_project
et-server  |     project_info = await fetch_project_version(app, owner, repo)
et-server  |   File "/usr/local/lib/python3.7/site-packages/etserver/getters.py", line 94, in fetch_project_version
et-server  |     app, GITHUB_ET_FILE.format(owner=owner, repo=repo)
et-server  |   File "/usr/local/lib/python3.7/site-packages/etserver/getters.py", line 10, in fetch_response
et-server  |     resp = await response.json()
et-server  |   File "/usr/local/lib/python3.7/site-packages/aiohttp/client_reqrep.py", line 1031, in json
et-server  |     headers=self.headers)
et-server  | aiohttp.client_exceptions.ContentTypeError: 0, message='Attempt to decode JSON with unexpected mimetype: ', url=URL('https://raw.githubusercontent.com/nipy/nipype/blob/master/.et')
```

Reply:
Just pushed a change that will hopefully fix this

Reply:
thanks!

Reply:
ahhh no, `app` is missing from the signature!